include/user_bench.h

```diff
@@ -74,8 +74,8 @@ namespace experimental {
 				void end_section(const std::string& name);
 
 				template <typename... Args>
-				void log(const std::string& format_string, Args&&... args) const {
-					spdlog::default_logger_raw()->log(spdlog::level::info, "[user] " + format_string, std::forward<Args>(args)...);
```

Comment:
What's the reasoning for this change?

Reply:
When building with C++20, fmtlib does compile-time format argument verification, so you can't just pass a `std::string` as the format argument, it either has to be an `fmt::format_string` which can be constexpr-constructed from a literal, or be wrapped in `fmt::runtime`. I went for the former.